tests/providers/apache/hive/transfers/test_s3_to_hive.py

```diff
@@ -107,8 +107,8 @@ def setUp(self):
                 self._set_fn(fn_bz2, '.bz2', False)
                 f_bz2_nh.writelines([line1, line2])
         # Base Exception so it catches Keyboard Interrupt
-        except BaseException as e:
-            logging.error(e)
+        except BaseException:
```

Comment:
This is rather strange - why do we need except/calling tearDown at all? tearDown should be called regardless (and BTW. we should convert it to fixture providing tmpdir (and cleaning it).

Reply:
Same shuffling and conversion to `pytest` here as well.